games-emulation/stella/stella-5.1.3.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils
+
+DESCRIPTION="Multi-platform Atari 2600 VCS Emulator"
+HOMEPAGE="https://stella-emu.github.io"
+SRC_URI="https://github.com/stella-emu/${PN}/releases/download/${PV}/${P}-src.tar.xz"
+
+LICENSE="GPL-2+ BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="joystick"
+
+RDEPEND="
+	media-libs/libsdl2[joystick?,opengl,video]
+	media-libs/libpng:0=
+	sys-libs/zlib
+"
+DEPEND="${RDEPEND}"
+DOCS="Announce.txt Changes.txt Copyright.txt README-SDL.txt Readme.txt Todo.txt"
+HTML_DOCS="docs"
```

Comment:
did you check what path this installs to? define as array as well, see previous review

Reply:
/usr/share/doc/${P}, right?

Reply:
did you test it?

Reply:
Yes, it installs where I expected it to.

Reply:
Not quite; `HTML_DOCS` is currently installed to `/usr/share/doc/stella-5.1.3/html/docs`, but it should be directly in `/usr/share/doc/stella-5.1.3/html`

Reply:
Just happened to have fixed that before I noticed your comment here.